code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -19,8 +19,6 @@
 		ChangeToHusk()
 	update_stat("updatehealth([reason])")
 	med_hud_set_health()
-	med_hud_set_status()
-	handle_hud_icons_health()
```

Comment:
the above two are called in `update_stat` well, `handle_hud_icons_health` is replaced with `update_health_hud`